mobile/ios/device/ios-device-file-system.ts

```diff
@@ -72,19 +72,19 @@ export class IOSDeviceFileSystem implements Mobile.IDeviceFileSystem {
 
 	public deleteFile(deviceFilePath: string, appIdentifier: string): void {
 		let houseArrestClient: Mobile.IHouseArrestClient = this.$injector.resolve(iOSProxyServices.HouseArrestClient, {device: this.device});
-		let afcClientForContainer = houseArrestClient.getAfcClientForAppContainer(appIdentifier);
-		afcClientForContainer.deleteFile(deviceFilePath);
+		let afcClientForAppDocuments = houseArrestClient.getAfcClientForAppDocuments(appIdentifier);
+		afcClientForAppDocuments.deleteFile(deviceFilePath);
```

Comment:
This code will not work in NativeScript CLI. The files there are always inside `/Library/Application Support/` directory, so this code will not be able to delete them. 
Maybe we should check the passed `deviceFilePath` and decide if we should use `getAfcClientForAppDocuments` or `getAfcClientForAppContainer`